app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
According to the checklist, you should avoid using an extra `else` statement. In your case, you don't have an `else` statement, but you have an equivalent structure. You return 0 if the `if` condition is not met. Instead, you could directly return 0 at the end of the function, and it will be returned if `self.is_hungry` is `False`. This makes the code cleaner and easier to read.